src/Resources/app/storefront/test/filter-category-select.plugin.test.js

```diff
@@ -2,15 +2,18 @@
  * @jest-environment jsdom
  */
 
-import FilterCategorySelectPlugin from "../src/js/filter-category-select.plugin";
+import FilterCategorySelectPlugin from '../src/js/filter-category-select.plugin';
+import FilterCategorySelectElement from './filter-category-select-helper';
+import Iterator from 'src/helper/iterator.helper';
 import ListingPlugin
-  from "../../../../../../../../vendor/shopware/platform/src/Storefront/Resources/app/storefront/src/plugin/listing/listing.plugin";
+    from '../../../../../../../../vendor/shopware/platform/src/Storefront/Resources/app/storefront/src/plugin/listing/listing.plugin';
 
 describe('filter-category-select.plugin.js', () => {
-  let filterCategorySelectPlugin;
+    let filterCategorySelectPlugin;
+    let filterCategorySelectElement;
 
-  beforeEach(() => {
-    window.csrf = {
+    beforeEach(() => {
+        window.csrf = {
```

Comment:
Check the formatting here too. Please fix the format issues everywhere where needed.